### PR TITLE
Added Risk Matrix to statistics

### PR DESCRIPTION
**Pull request by SyncInProgress**

### CODE REVIEW

File: src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.editor.Editor;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.select.Select;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import org.coffeemine.app.spring.data.Risk;
+import org.coffeemine.app.spring.view.View;
+
+import java.util.*;
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java]: import * nono

File: src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+public class RiskMatrix extends View {
+
+    private List<Risk> riskList;
+
+    public RiskMatrix(){
+
+        riskList = new ArrayList<>();
+        riskList.add(new Risk ("No or poor business case", "Commercial", "Select", "Select"));
```
Comment (RuthgerD) [inline src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java]: should be done more efficiently but cant blame ya for doing it like this

Comment (AeroStun) [reply src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java]: Too bad Java doesn't have `constexpr`

File: src/main/java/org/coffeemine/app/spring/data/Risk.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package org.coffeemine.app.spring.data;
+
+public class Risk {
```
Comment (AeroStun) [inline src/main/java/org/coffeemine/app/spring/data/Risk.java]: Pretty sure this should be `JSONSerializable` and `NO2Serializable`

File: src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java [hunk 4]
```diff
@@ -148,11 +159,78 @@ public RiskMatrix(){
...
+        filterRow.getCell(likelihoodColumn).setComponent(likelihoodFilter);
+        likelihoodFilter.setSizeFull();
+        likelihoodFilter.setPlaceholder("Search");
+
         this.add(riskMatrix);
 
     }
 
+    private Button createRemoveButton(Grid<Risk> grid, Risk risk) {
+        @SuppressWarnings("unchecked")
```
Comment (AeroStun) [inline src/main/java/org/coffeemine/app/spring/components/RiskMatrix.java]: That's always fun

File: src/main/java/org/coffeemine/app/spring/data/Risk.java [hunk 5]
```diff
@@ -1,19 +1,31 @@
...
     }
 
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
```
Comment (AeroStun) [inline src/main/java/org/coffeemine/app/spring/data/Risk.java]: If not used anywhere, remove

**Pull request merged**